Assets/Scripts/Importing/Items/Definitions/VehicleDef.cs

```diff
@@ -20,6 +20,55 @@ public enum VehicleType
     [Section("cars")]
     public class VehicleDef : Definition, IObjectDefinition
     {
+        public struct CompRulesUnion
```

Comment:
this struct has 48 bytes, but it could have only 4

but it's fine, I can fix this easily, if it's a problem for you :)

Reply:
Do you mean to declare all of those members as getters, and extract there al those calculations?
Something like:
`public bool nExtraAComp
{
    get
    {
        return (nExtraA & 0x0FFF) >> 0;
    }
}`

Reply:
yeah, I think that would be the easiest approach